auction-server/src/auction.rs

```diff
@@ -1589,34 +1601,36 @@ impl ChainStore for ChainStoreSvm {
             .try_into()
             .map_err(|_| anyhow!("Invalid svm signature"))?;
         if bids.len() != 1 {
```

Comment:
a small nit: the function that gets the `bids` to pass into `get_bid_results` is `bids_for_submitted_auction`. technically, `bids` will have length greater than 1 iff multiple bids in _different_ auctions have the same tx hash. For this reason, it may be good idea to rename the `bids_for_submitted_auction` function to reflect this, e.g. maybe to `bids_matching_tx_hash`?

Reply:
What about evm?